Remove a conditional that decides whether `quarto-preview.js`, the code which
handles hot-reloading when using `quarto preview`, should be built. In our
case, it should be, but the conditional was deciding not to.

IMPORTANT: This patch is temporary. A fix will be applied upstream to enable us
to remove this patch. See: 
https://github.com/conda-forge/quarto-feedstock/pull/36#issuecomment-1947515694 

diff --git a/src/webui/quarto-preview/build.ts b/src/webui/quarto-preview/build.ts
index 625cc7910..728afb060 100644
--- a/src/webui/quarto-preview/build.ts
+++ b/src/webui/quarto-preview/build.ts
@@ -11,25 +11,7 @@ try {
 }
 
 // check if any of our repo files have a later time
-let build = false;
-try {
-  const command = new Deno.Command("git", { args: ["ls-files"] });
-  const cmdOutput = await command.output();
-  if (cmdOutput.success) {
-    const output = new TextDecoder().decode(cmdOutput.stdout);
-    const files = output.split("\n").filter((line) => line.length > 0);
-    build = files.some((file) =>
-      Deno.statSync(file).mtime!.valueOf() > jsBuildTime
-    );
-  } else { 
-    // not in a git repo, rebuild
-    build = true;
-  }
-} catch {
-  // git not installed, rebuild
-  build = true;
-}
-
+let build = true;
 if (build) {
   const buildCommand = new Deno.Command(Deno.execPath(), {
     args: ["task", "build"],
